src/modules/data/blocks/ticket/constants.js

```diff
@@ -4,6 +4,7 @@ export const WOO = 'woo';
 export const RSVP = 'rsvp';
 
 export const RSVP_CLASS = 'Tribe__Tickets__RSVP';
+export const TEC_TICKETS_COMMERCE_MODULE_CLASS = 'TEC\\Tickets\\Commerce\\Module';
```

Comment:
I would avoid `TEC_` prefix on this case.

Reply:
Thanks! Constant was renamed